src/index/Vocabulary.h

```diff
@@ -491,85 +309,76 @@ class Vocabulary {
   static void prefixCompressFile(const string& infile, const string& outfile,
                                  const vector<string>& prefixes);
 
-  // ___________________________________________________________________
-  void setCaseInsensitiveOrdering(bool ignoreCase) {
-    _caseComparator = StringSortComparator(ignoreCase);
+  void setLocale(const std::string& language, const std::string& country,
+                 bool ignorePunctuation) {
+    _caseComparator = ComparatorType(language, country, ignorePunctuation);
   }
 
-  // ___________________________________________________________________
-  bool isCaseInsensitiveOrdering() const {
-    return _caseComparator.isIgnoreCase();
+  // _____________________________________________________________________
+  const ComparatorType& getCaseComparator() const { return _caseComparator; }
+
+  /// returns the range of IDs where strings of the vocabulary start with the
+  /// prefix according to the collation level the first Id is included in the
```

Comment:
`///` vs `//!`, I think we should remove the latter when we edit a comment, I've only ever seen that in QLever

Reply:
I agree. I have adapted all the comments in the StringSortComparator.h.
As for the vocabulary, there are two big changes I am planning to make. Is it ok if I will rework the comments in these following PRs, I will have to touch many functions there anyways